.github/workflows/build.yml

```diff
@@ -0,0 +1,102 @@
+name: moonlight-android
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: android-actions/setup-android@v3
+      with:
+        cmdline-tools-version: 9477386
+
+    - uses: actions/setup-java@v4
+      with:
+        java-version: 21
+        distribution: 'adopt'
+
+    - name: Set up environment
+      run: |
+        echo "ANDROID_NDK_HOME=$ANDROID_NDK_HOME" >> $GITHUB_ENV
```

Comment:
This line is not actually doing anything?

Reply:
Correct, I have tested it, it is not necessary.